snappy.go

```diff
@@ -6,7 +6,7 @@
 // It aims for very high speeds and reasonable compression.
 //
 // The C++ snappy implementation is at https://github.com/google/snappy
-package snappy // import "github.com/golang/snappy"
```

Comment:
If you're working on a local fork, you can check that out manually to the correct location (`$GOPATH/src/github.com/golang/snappy`) even if the remote is e.g. github.com/AlasdairF/snappy. `go get` is a convenience; sometimes it doesn't fit your use case and you need to use `git clone`.

This import comment is intentional, and should be left in.